### PR TITLE
Necropolis Tendrils give off more light, and will remain after collapsing to light the chasm around them

### PR DESCRIPTION
**Pull request by ChangelingRain**
:cl: Joan
tweak: Necropolis Tendrils give off more light, and will remain after collapsing to light the chasm around them.
/:cl:

Before collapse

Before chasm forms

After chasm forms(the legion pushed the chest off the tendril, the chest does not normally despawn)

### CODE REVIEW

**Comment by obscolene** [comment]
revert the meson changes and we wouldnt have this problem

**Comment by Kromgar** [comment]
Can you make them brighter? Its still pretty dark

**Comment by ChangelingRain** [comment]
Yes; is this good?

**Comment by Kromgar** [comment]
@ChangelingRain Yeah thats way more noticeable

File: code/modules/mob/living/simple_animal/hostile/mining_mobs.dm [hunk 1]
```diff
@@ -1030,29 +1038,44 @@
...
+	playsound(T,'sound/effects/explosionfar.ogg', 200, 1)
+	for(var/mob/M in range(7,T))
+		shake_camera(M, 15, 1)
+	for(var/turf/F in circlerangeturfs(T, 2) - T)
+		F.TerraformTurf(/turf/open/chasm/straight_down/lava_land_surface)
+
+/obj/effect/light_emitter/tendril
+	set_luminosity = 4
+	set_cap = 3
+	light_color = LIGHT_COLOR_LAVA
```
Comment (lzimann) [inline code/modules/mob/living/simple_animal/hostile/mining_mobs.dm]: why tho

Comment (ChangelingRain) [inline code/modules/mob/living/simple_animal/hostile/mining_mobs.dm]: They're orange!

**Comment by Repukan** [comment]
Make it dimmer, miners and ghost roles get lights

**Comment by FantasticFwoosh** [comment]
It doesn't matter, you can put rods over chasms anyway and it changes them to lava & metal rod turf to path to the broken tendril which appears to not even have any valuable second loot.

Forgive if i've mistaken the code but by `GIBTONITE=INERT` does that imply that if i path over there using rods & metal floor tiles, i will be able to activate the tendril to explode/harvest for gibtonite as per a not thought out feature?

**Comment by KorPhaeron** [comment]
>remain
>after collapsing

???

**Comment by KorPhaeron** [comment]
More importantly this removes the risk of grabbing the chest entirely

**Comment by ChangelingRain** [comment]
entirely??

**Comment by ChangelingRain** [comment]
maybe if you have a flight potion sure

**Pull request closed without merging**